src/overlays/actors/ovl_En_Sob1/z_en_sob1.c

```diff
@@ -8,7 +14,68 @@ void EnSob1_Init(Actor* thisx, GlobalContext* globalCtx);
 void EnSob1_Destroy(Actor* thisx, GlobalContext* globalCtx);
 void EnSob1_Update(Actor* thisx, GlobalContext* globalCtx);
 
-#if 0
+void EnSob1_DrawZoraShopkeeper(Actor* thisx, GlobalContext* globalCtx);
+void EnSob1_DrawGoronShopkeeper(Actor* thisx, GlobalContext* globalCtx);
+void EnSob1_DrawBombShopkeeper(Actor* thisx, GlobalContext* globalCtx);
+
+void EnSob1_Idle(EnSob1* this, GlobalContext* globalCtx);
+void EnSob1_Blink(EnSob1* this);
+void EnSob1_Walk(EnSob1* this, GlobalContext* globalCtx);
+s16 EnSob1_GetXZAngleAndDistanceSqToPoint(Path* path, s32 pointIdx, Vec3f* pos, f32* distSq);
+void EnSob1_Walking(EnSob1* this, GlobalContext* globalCtx);
+void EnSob1_Hello(EnSob1* this, GlobalContext* globalCtx);
+void EnSob1_StartShopping(GlobalContext* globalCtx, EnSob1* this);
+void EnSob1_SetupIdle(EnSob1* this, GlobalContext* globalCtx);
+void EnSob1_FaceShopkeeper(EnSob1* this, GlobalContext* globalCtx);
+void EnSob1_LookToShelf(EnSob1* this, GlobalContext* globalCtx);
+void EnSob1_EndingInteraction(EnSob1* this, GlobalContext* globalCtx);
+void EnSob1_BrowseShelf(EnSob1* this, GlobalContext* globalCtx);
+s32 EnSob1_ReturnItemToShelf(EnSob1* this);
+void EnSob1_TalkingToShopkeeper(EnSob1* this, GlobalContext* globalCtx);
+void EnSob1_SelectItem(EnSob1* this, GlobalContext* globalCtx);
+void EnSob1_LookToShopkeeperFromShelf(EnSob1* this, GlobalContext* globalCtx);
+void EnSob1_BuyItemWithFanfare(EnSob1* this, GlobalContext* globalCtx);
+void EnSob1_CanBuy(EnSob1* this, GlobalContext* globalCtx);
+void EnSob1_CannotBuy(EnSob1* this, GlobalContext* globalCtx);
+void EnSob1_SetupItemPurchased(EnSob1* this, GlobalContext* globalCtx);
+void EnSob1_ResetItemPosition(EnSob1* this);
+void EnSob1_ContinueShopping(EnSob1* this, GlobalContext* globalCtx);
+s32 EnSob1_TestCancelOption(EnSob1* this, GlobalContext* globalCtx, Input* input);
+s32 EnSob1_TakeItemOffShelf(EnSob1* this);
+void EnSob1_InitZoraShopkeeper(EnSob1* this, GlobalContext* globalCtx);
+void EnSob1_InitGoronShopkeeper(EnSob1* this, GlobalContext* globalCtx);
+void EnSob1_InitBombShopkeeper(EnSob1* this, GlobalContext* globalCtx);
+void EnSob1_InitShop(EnSob1* this, GlobalContext* globalCtx);
+
+extern UNK_TYPE D_0401F740;
+extern UNK_TYPE D_0401F8C0;
+extern UNK_TYPE D_0401F7C0;
+extern AnimationHeader D_06009120;
+extern AnimationHeader D_06008268;
+extern FlexSkeletonHeader D_0600D208;
+extern AnimationHeader D_0600078C;
+extern AnimationHeader D_060087BC;
+extern FlexSkeletonHeader D_06011AC8;
+extern AnimationHeader D_060000FC;
+extern FlexSkeletonHeader D_06009220;
+extern Gfx D_06000970[];
+extern UNK_TYPE D_06005458;
+extern UNK_TYPE D_060050A0;
+extern UNK_TYPE D_060058A0;
+extern UNK_TYPE D_060060A0;
+extern UNK_TYPE D_06010438;
+extern UNK_TYPE D_06010C38;
+extern UNK_TYPE D_06011038;
+
+const char filname1[] = "../z_en_soB1.c"; // Unused
+const char filname2[] = "../z_en_soB1.c"; // Unused
```

Comment:
Typo?

Reply:
Yeah, though I'm going to change the implementation to use `(void)"../z_en_soB1.c"` like `en_ossan`